Nasal/engine.nas

```diff
@@ -291,12 +389,25 @@ setlistener("/controls/switches/starter", func {
             primerTimer.stop();
         }
     }
+    
+    # sorry - had to hack this to prevent coughing on startup due to the oil pressure simulation. Maybe this can be used elsewhere
+     if (getprop("/controls/engines/active-engine") == 0)
+       var rpm = getprop("/engines/engine[0]/rpm");
```

Comment:
Why don't you just do `var rpm = getprop("/engines/active-engine/rpm");` here?

Reply:
Because of "Nasal runtime error: nil used in numeric context
  at C:/Users/redpa/Documents/FlightGear/Custom Aircraft/c172p/Nasal/engine.nas, line 399
  called from: C:/Users/redpa/Documents/Programs/FlightGear 2018.1.1/data/Nasal/globals.nas, line 119"